server/mailet/integration-testing/src/test/resources/smtpserver.xml

```diff
@@ -44,6 +44,9 @@
         <handlerchain>
             <handler class="org.apache.james.smtpserver.fastfail.ValidRcptHandler"/>
             <handler class="org.apache.james.smtpserver.CoreCmdHandlerLoader"/>
+            {{#hooks}}
+            <handler class="{{hookFqcn}}"/>
+            {{/hooks}}
```

Comment:
you cannot generate a configuration object?

Reply:
No, that's out of scope, and sadly not "easy" to do. (spoiler this means reworking concequently all Protocols (IMAP + SMTP + LMTP + FetchMail + ManageSieve).

Thank you inheritance.